src/dsp_tools/resources/start-stack/docker-compose.yml

```
@@ -28,7 +28,7 @@ services:
 
   sipi:
     # on the verge of every deployment (fortnightly), take the same tag as DSP-API
-    image: daschswiss/knora-sipi:v31.7.0
+    image: daschswiss/knora-sipi:v31.8.0-1-g9805e5d
```

Why a dirty commit? Usually we take the same as "api":

```suggestion
    image: daschswiss/knora-sipi:v31.8.0
```

---

Ah, I wasn't sure where that came from, so I opted for the last just to be sure. Yes, let's take the stable.